docs/hosting-a-server-with-northstar/quick-start.md

```diff
@@ -48,8 +57,8 @@ sv_cheats 1; script GameRules_ChangeMap( "mp_forwardbase_kodai", "ctf" ); sv_che
 ```
 
 replace `mp_forwardbase_kodai` and `ctf` with your desired map and gamemode.\
-The list of maps can be found [here](dedicated-server/#maps).\
-The list of gamemodes [here](dedicated-server/#gamemodes).
+The list of maps can be found [here](./server-settings/file-names.md#maps).\
+The list of gamemodes [here](./server-settings/file-names.md#vanilla).
```

Comment:
More stuff missed in #218? x_x

Can you just quickly PR that change separately?

Reply:
Fixed the links myself now via #229

Make sure to update PR to resolve merge conflicts.

Reply:
I guess its resolved LOL